libexec/trick/make_makefile_swig

```diff
@@ -218,6 +218,14 @@ SWIG_OBJECTS = \$(subst .cpp,.o,\$(SWIG_SRC)) $swig_src_dir/init_swig_modules.o
 \$(S_MAIN): \$(SWIG_OBJECTS)
 
 LINK_LISTS += \$(LD_FILELIST)build/py_link_list
+
+# trick.zip ====================================================================
+
+trick.zip: \$(SWIG_SRC)
+\t\$(call ECHO_AND_LOG,zip -rq trick trick)
+\t\$(call ECHO_AND_LOG,rm -rf trick)
```

Comment:
I don't see the rule to pre-compile all the `*.pyc` files, is it somewhere else?